heroic-dist/src/test/java/com/spotify/heroic/AbstractClusterQueryIT.java

```diff
@@ -17,9 +18,12 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.verifyNoMoreInteractions;
 
+
```

Comment:
would be nice formatting to remove the extra spacing if you're fixing other things but not a big deal

Reply:
I think this empty line separate static import statements from the other import.  But I can remove the line if that is preferable.